stockflux-securities-master/src/components/reusableComponents/ToggleSwitch.js

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+import "./ToggleSwitch.css";
+
+const TextField = ({ value, id, checked, onChange }) => {
+  return (
+    <div>
+      <input
+        value={value}
+        id={id}
+        checked={checked}
+        type="checkbox"
+        className="input-form-toggle"
+        onChange={onChange}
+      />
+      <label className="toggle-switch" htmlFor={id} />
```

Comment:
This label is empty, should it have a value or is it not meant to be here?

Reply:
This label is empty because it's simply there for CSS styling purposes. It does it this way in a tutorial I found on how to make a toggle switch.

Reply:
Just spoke to you about this and had a quick demo so this is fine